shared/js/template.js

```diff
@@ -113,25 +113,43 @@
    *                          - safe, a list of properties that contain
    *                          HTML that is known and are
    *                          "known" to ignore.
-   */
-  Template.prototype.interpolate = function(data, options) {
-    // This _should_ be rewritten to use Firefox's support for ES6
-    // default parameters:
-    // ... = function(data, options = { safe: [] }) {
-    //
-    options = options || {};
-    options.safe = options.safe || [];
+  */
+  Template.prototype.interpolate = function(data = {}, options = { safe: [] }) {
+    var safeList = options.safe;
+    var hasSafeList = Boolean(safeList);
```

Comment:
nit: we usually use `!!safeList` for this. But to be honest I don't think we need this boolean :) you can just use `safeList` instead